prediction_market_agent/agents/microchain_agent/app.py

```diff
@@ -255,15 +265,28 @@ def get_function_bullet_point_list(agent: Agent, model: str) -> str:
     else:
         st.markdown("The agent is not initialized yet.")
 
+
 # Placeholder for the agent's history
-history_container = st.container()
+with st.container() as history_container:
```

Comment:
If `container` is created and used and the same place, there is no need for the container, is there?